src/test/integ/buffer-write-cmd.test.ts

```diff
@@ -15,6 +16,22 @@ import {
 
 describe("BufWriteCmd integration", () => {
     const testFiles: Uri[] = [];
+    const cleanupCallbacks: (() => Promise<void>)[] = [];
+
+    const cleanupFileCallback = async (uri: Uri) => {
+        try {
+            await workspace.fs.delete(uri);
+        } catch {
+            // ignore
```

Comment:
is this expected to happen frequently? if not, usually worth at least logging a warn-level message.